### ISSUE TITLE
Accessing a secret with its url after its already been opened throws error in logs.

### ISSUE THREAD

**Issue by Stan805** [theCalcaholic/nextcloud-secrets#44]
Title basically. Its a bit annoying and I think it shouldn't throw errors in the logs. My NC version is 28.0.2

**Comment by theCalcaholic** [comment]
Do you mean this error message?

```
[Tue Jun 11 23:25:32.346281 2024] [php:notice] [pid 791] [client 192.168.21.5:39912] Did expect one result but found none when executing: query "SELECT * FROM `*PREFIX*secrets` WHERE (`uuid` = :dcValue1) AND (`encrypted` IS NOT NULL) AND (`expires` > :dcValue2)";
```

I agree, that this probably shouldn't be logged...

**Comment by theCalcaholic** [comment]
A fix is in the nightly branch and will soon be released

**Comment by theCalcaholic** [comment]
Fix is included in v2.0.0
